erpnext/templates/pages/help.html

```diff
@@ -31,6 +72,7 @@ <h2>{{ section["name"] }}</h2>
 </div>
 {% endfor %}
 
+<!-- {% if topics %}
```

Comment:
Remove commented code. Also if you are planning to hide the forum section, just remove its code from `help.py` and the doctype

Reply:
OK